### ISSUE TITLE
FeatureInfo: Render Markdown into React components, add support for conditional rendering

### ISSUE THREAD

**Issue by jacobwod** [hajkmap/Hajk#669]
# Problem
The current implementation has two major flaws:
1. The markdown can not contain any [tags]. This is due to the _shortcode_ feature in FeatureInfo that basically sees everything between [ and ] as a tag. This conflicts with Markdown's Links and Images syntax.
2. The resulting HTML is plain HTML, whereas we'd like to use MUI components on some ocassions. For example, a table should not parse into `<table>` but rather the MUI Table Component, to ensure correct rendering, dark mode support etc.

# Suggestions
1. **I will initially remove the _shortcode_ functionality.** We could look into making something similar, but it must not conflict with Markdown syntax.
2. I'll see if our Markdown parser can be extended to render MUI Components. A quick search gave [this promising result](https://marked.js.org/using_pro). Update: using `react-markdown` makes this possible.
3. Implement some kind of conditional rendering. It could be used e.g not to render empty fields.

# Currently implemented
- Fixed problem 1: Markdown can use all []-tags.
- Fixed problem 2: We return React components
- Added support for suggestion 3, see below.

## Example of conditional rendering
```
<if {foo}>Will show if 'foo' is truthy</if>

<if {foo}=bar>Will show if 'foo' evaluates to 'bar'</if>
```

**Comment by jacobwod** [comment]
See https://github.com/hajkmap/Hajk/tree/feature/669-new-featureinfo

**Comment by jacobwod** [comment]
This is almost done, though it's been a while since I looked into this last time. AFAIR, everything works as expected, except the new DocumentHandler links, which need to be sent as Components. 

For the purpose of documentation, can @sweco-seqtob show me an example of code for Infoclick box (in admin) that is needed in order to render a DocumentHandler link? (I finally got DocumentHandler running this week so I can set it up accordingly now.)
